tests/code/test_volunteers.py

```diff
@@ -72,7 +73,7 @@ def test_updateEventParticipants():
     with pytest.raises(DoesNotExist):
         EventParticipant.get(EventParticipant.user=="partont", EventParticipant.event==3)
 
-    participantData = {'inputHours_partont':100, 'checkbox_partont':"on", 'event':3, 'username1': 'partont'}
+    participantData = ImmutableMultiDict({'inputHours_partont':100, 'checkbox_partont':"on", 'event':3, 'username': 'partont'})
     volunteerTableUpdate = updateEventParticipants(participantData)
     assert volunteerTableUpdate == True
 
```

Comment:
We should update this test (or one of the other ones) to test that having multiple usernames in the ImmutableMultiDict works.